api/argo-server/swagger.json

```diff
@@ -5516,31 +5516,5 @@
         }
       }
     }
-  },
```

Comment:
I am not sure what caused these changes.

Reply:
just check out from master to fix